src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/TopErrorReportViewModel.swift

```diff
@@ -27,11 +27,7 @@ class TopErrorReportViewModel {
 					text: AppStrings.ErrorReport.description1,
 					accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.topBody
 				),
-				.link(
-					text: AppStrings.ErrorReport.faq,
-					url: URL(string: AppStrings.ErrorReport.faqURL),
-					accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.faq
-				)
+				.link(placeholder: AppStrings.ErrorReport.faq, link: AppStrings.ErrorReport.faqURL, font: .subheadline, style: .subheadline, accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.faq)
```

Comment:
Can you follow the pattern like it was before? It was better to read the code.

Reply:
Thanks, Done!